### PR TITLE
Update the translating

### PR DESCRIPTION
**Pull request by VikingBO**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ❌
| Fixed issues  | ❌

Just translating

### CODE REVIEW

File: docs/guide-ru/security-authorization.md [hunk 1]
```diff
@@ -224,6 +224,10 @@ return [
 ];
 ```
 
+> Примечание: Если вы используете версию yii2-basic-app, кроме конфигурации веб приложения `config/web.php` подключение компонента
```
Comment (samdark) [inline docs/guide-ru/security-authorization.md]:
```suggestion
> Примечание: Если вы используете шаблон проекта basic, компонент `authManager` необходимо настроить как в `config/web.php`, так и в
```

File: docs/guide-ru/security-authorization.md [hunk 2]
```diff
@@ -224,6 +224,10 @@ return [
...
+> `authManager` нужно также указать в [конфигурации консольного приложения](tutorial-console.md#configuration) `config/console.php`.
```
Comment (samdark) [inline docs/guide-ru/security-authorization.md]:
```suggestion
> [конфигурации консольного приложения](tutorial-console.md#configuration) `config/console.php`.
```

File: docs/guide-ru/security-authorization.md [hunk 3]
```diff
@@ -224,6 +224,10 @@ return [
...
+> В версии yii2-advanced-app класс `authManager` нужно подключить один раз в `common/config/main.php`.
```
Comment (samdark) [inline docs/guide-ru/security-authorization.md]:
```suggestion
> При использовании шаблона проекта advanced `authManager` достаточно настроить единожды в `common/config/main.php`.
```

**Review by samdark — CHANGES_REQUESTED** [review]

**Comment by VikingBO** [comment]
I don't understand what I should do next to accept the request

**Comment by samdark** [comment]
Take suggestions into account, not just close them.

**Review by VikingBO — COMMENTED** [review]
upd changes

**Comment by VikingBO** [comment]
And how i can restart tests?

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**